src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -42,4 +44,8 @@ const ContentInsightsSummary = ({ error, graphData, isLoading, previousPeriodCou
     return <div className="ContentInsightsSummary">{renderContentAnalyticsSummary()}</div>;
 };
 
+ContentInsightsSummary.propTypes = {
```

Comment:
Do we need to add this if we already have typescript for typing?

Reply:
I was getting this `"status" is missing in props validation` warning. I found this to be the only way to remove it